spec/requests/invitations_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+RSpec.describe "/users/invitation", type: :request do
+  let(:user) { create(:user, registered: false) }
+
+  describe "PUT /users/invitation" do
+    it "updates the user's 'registered' column upon invite acceptance" do
+      user.invite!
+      # image_path = Rails.root.join("spec/support/fixtures/images/image1.jpeg")
+      # user = User.invite!(email: "hey#{rand(1000)}@email.co",
+      #                     name: "Roger #{rand(1000)}",
+      #                     username: "rogerabc",
+      #                     remote_profile_image_url: Rack::Test::UploadedFile.new(image_path, "image/jpeg"),
+      #                     saw_onboarding: false,
+      #                     editor_version: :v2,
+      #                     registered: false)
+      # remote_profile_image_url: Rack::Test::UploadedFile.new(image_path, "image/jpeg"),
+      #   user.accept_invitation!
+      #   user.accept_invitation!(invitation_token: params[user.invitation_token])
```

Comment:
The issue that I am currently running into is that I cannot stub out the invite process (sending a user an invitation) in the specs. I have looked into this extensively and have gotten help for multiple team members across multiple teams and have _still_ be unable to get this to work correctly. 

Currently, this commented out code mirrors the code that sends a user an invitation that can found in `/admin/invitations_controller.rb` in addition to the [`devise_invitatable` documentation](https://github.com/scambra/devise_invitable) on how to accept an invitation. 

The code itself seems to be working great, but because I am unable to get this spec to both work and pass, I cannot be too sure -- this is where I am blocked and could _really_ use some help, please and thank you in advance, team!

cc: @Ridhwana

Reply:
**_Update_**: we've decided to temporarily comment this spec out so that it doesn't hold this PR up any longer. I've added a comment to the spec that references the Internal issue that we opened so that we can circle back around to this spec when we have a bit more time.

Reply:
@juliannatetreault given that I have another PR which happened to add the same file (#10934) why don't we delete this file altogether for now to avoid a conflict, and then we can add it in later?

IMO this PR is looking good and we can remove from draft at least to discuss on those terms.

Reply:
Sounds good to me and done, @benhalpern! 👍